includes/class-wc-admin-reports-sync.php

```diff
@@ -432,6 +432,26 @@ public static function orders_lookup_import_batch( $batch_number, $days, $skip_e
 	 * @return void
 	 */
 	public static function orders_lookup_import_order( $order_id ) {
+
+		$order = wc_get_order( $order_id );
```

Comment:
I think this is probably a good way of handling things.

But I'm also curious if you ran across any specific cases/context where we were able to get an order through `get_orders()` that resulted in `wc_get_order()` returning `false`.

Reply:
yes:

https://wordpress.org/support/topic/php-fatal-error-call-to-a-member-function-get_type-on-boolean/

Reply:
@psealock Maybe I'm missing something in that post, but I'm not seeing the reason why `wc_get_order()` returned false for the order ID.

It doesn't invalidate this PR, just curious if there's something else happening that we should be aware of.

Reply:
Good point. This will be something to keep an eye on. Looking at the `get_orders` function in the same file doesn't look like it would produce anything odd.